dask/dataframe/tests/test_accessors.py

```diff
@@ -233,7 +233,9 @@ def test_str_accessor_noexpand():
 
 
 def test_str_accessor_expand():
-    s = pd.Series(["a b c d", "aa bb cc dd", "aaa bbb ccc dddd"])
+    s = pd.Series(
+        ["a b c d", "aa bb cc dd", "aaa bbb ccc dddd"], index=["row1", "row2", "row3"]
```

Comment:
Just to confirm: an object-dtype index produces the same buggy behavior as a DatetimeIndex?

Reply:
That's correct. I could make a more extensive (separate) test for it, but I felt this was sufficient. WDYT?

Reply:
Nope, no need. I think this covers everything we care about.